doc/nrf/libraries/caf/sensor_manager.rst

```diff
@@ -160,6 +165,52 @@ To use the sensor trigger, complete the following steps:
 .. note::
     |only_configured_module_note|
 
+Enabling passive power management
+=================================
+
+The |sensor_manager| can react to ``power_down_event`` and ``wake_up_event``.
+This functionality is called passive power management.
+
+To use the passive power management in |sensor_manager|, complete the following steps:
+
+1. Enable :kconfig:`CONFIG_CAF_SENSOR_MANAGER_PM` option.
+   If a sensor of your choice supports trigger functionality,its enabled and configured by you, there is nothing more to do.
+   If you did not enable trigger functionality for your sensor, continue with next steps.
+#. Extend the module configuration file by adding :c:member:`sensor_config.suspend_pm_state` in an array of :c:struct:`sensor_config`.
```

Comment:
Note that this field is interpreted only if the :trigger field is not set.

Reply:
Its mentioned in implementation details. This section is only for overview of the feature and how to enable it, no implementation details.

Reply:
But here are configuration details. Ones may just read how to configure it and does not even look into implementation details section.

Reply:
Im about to tune `Enabling ...` sections a bit so it at least point to implementation details.

Reply:
And briefly say what it does, but very briefly. More would be found in implementation details.